### ISSUE TITLE
Deactivate "Save" button in metadata editor until data changes

### ISSUE THREAD

**Issue by solth** [kitodo/kitodo-production#4438]
Similar to how the "Save" button is deactivated on "Project edit", "User edit" etc. pages until the some data has been changed by the user, the "Save" button in the metadata editor should also be deactivated until there is something to save (as an indication to the user that there are unsaved changes)

**Comment by matthias-ronge** [comment]
Note that there may be changes that occur when the metadata editor is started, namely when new images have been found and inserted into the workpiece. In that case you should be able to save immediately. Just to mention it.

**Comment by andre-hohmann** [comment]
I can support @matthias-ronge hint. 
The feature is "Read in pagination from images" which is done automatically and is absolutely necessary to write the references in the METS-file. Many users wish, that the references are written independently from the editor. If these references cannot be saved, the METS-files are incomplete.

**Comment by matthias-ronge** [comment]
> Many users wish, that the references are written independently from the editor. 

There is an issue for this: #3333

**Comment by solth** [comment]
> Note that there may be changes that occur when the metadata editor is started, namely when new images have been found and inserted into the workpiece. In that case you should be able to save immediately. Just to mention it.

Good point. Perhaps it would be useful to have a small info dialog in the metadata editor (or at least a growl message) when new images have been found to explicitly inform the user that there are new unsaved changes?
